app/views/notifications/_tagadjustment.html.erb

```diff
@@ -1,28 +1,31 @@
-<div class="small-pic">
-  <img src="https://res.cloudinary.com/practicaldev/image/fetch/s--LnVw15KE--/c_fill,f_auto,fl_progressive,h_90,q_auto,w_90/https://thepracticaldev.s3.amazonaws.com/uploads/user/profile_image/31047/af153cd6-9994-4a68-83f4-8ddf3e13f0bf.jpg"
-       alt="Sloan, the sloth mascot">
-</div>
-<div class="content notification-content comment-content">
-  <% data = notification.json_data %>
-  <% removal = data["adjustment_type"] == "removal" %>
-  Hey there 👋
-  <br /><br />
-  Moderators <%= removal ? "removed" : "added" %>  the
-  <b><a href="/t/<%= data["tag_name"] %>" style="font-weight: 900">#<%= data["tag_name"] %></a></b> tag <%= removal ? "from" : "to" %> your post
-  <a href="<%= data["article"]["path"] %>"><%= h(data["article"]["title"]) %></a>.
-  <br /><br />
-  <%= removal ? "Certain tags have special submission guidelines, and mods need to adjust tags to best fit the community." : "Mods felt this tag represented your article." %>
-  <br /><br />
-  <div class="comment-text">
-    <div style="font-size:0.8em;margin-top: -3px;font-weight: 900;margin-bottom: 5px;">Reason for action</div>
-    <%= data["reason_for_adjustment"] %>
+<div class="crayons-card notification">
+  <span class="crayons-avatar crayons-avatar--xl shrink-0">
+    <img src="https://res.cloudinary.com/practicaldev/image/fetch/s--LnVw15KE--/c_fill,f_auto,fl_progressive,h_90,q_auto,w_90/https://thepracticaldev.s3.amazonaws.com/uploads/user/profile_image/31047/af153cd6-9994-4a68-83f4-8ddf3e13f0bf.jpg" class="crayons-avatar__image" alt="Sloan, the sloth mascot" width="48" height="48">
+  </span>
+
+  <div class="notification__content">
+    <% data = notification.json_data %>
+    <% removal = data["adjustment_type"] == "removal" %>
+      <h3 class="mb-2">Hey there 👋</h3>
```

Comment:
Since most of the other notification cards have `color-base-90` headings, I think we can make it uniform here as well. This is static text and not a link, but this is something we can do to create most consistency around the cards.